## dsRNA-seq

**Technology**

dsRNA-seq stands for double stranded RNA sequencing. It's a method to learn about the secondary and tertiary structure of RNA in vitro. We use different RNAses that digest RNA. The resulting fragments are reverse-transcribed to cDNA using reverse transcriptase. Since single-stranded RNA is digested by a different RNase than double-stranded RNA, we can then compare digestion results of different RNases and make conclusions about the structure of RNA.

**Application**

RNAs are responsible for many different tasks, such as protein synthesis and regulation of gene expression.
Hence, understanding the correct structure of RNA is paramount, because its functionality is intimately linked to its 3D structure. We have a lack of data on how RNA forms its secondary structure from its base-pairs interacting.

**Statistics**

Some of the statistical methods employed include
* Geometric distribution-based approach to identify unusually long dsRNA molecules
* Poisson distribution-based statistical approach to identify regions of the genome that are significantly enriched in smRNA (small modulatory RNA that is double-stranded) molecules
*  Binomial-test to classify dsRNA clusters as intra- or inter-molecular by determining whether they demonstrate comparable amounts of reads from both strands (inter) or not (intra).

**Sources**

https://www.illumina.com/science/sequencing-method-explorer/kits-and-arrays/pars-dsrna-seq.html
https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1001141#pgen.1001141.s017

**Github Usernames**

@flavio87, @Cauawest, @Alexander-Syn, @ahmadrezauf
