
## FAIRE-seq

#### Short summary of the technology

The identification of nucleosome-depleted DNA is crucial for the analysis of regulatory elements. A method called FAIRE-seq  (Formaldehyde-Assisted Isolation of Regulatory Elements)  developed by Giresi et al. (2007) [^1] addressed this challenge by cross-linking chromatin and nucleosomes, followed by a sequencing of the open regions of DNA. FAIRE-seq has proven to be a powerful genome-wide approach for detecting functional in vivo regulatory elements involved in various mechanisms such as tumor development [^2], cell-type differentiation [^3] or chromatin organization [^4].

#### Some applications

- Discovery of open-chromatin,

- Identification of regulatory elements,

- Identification of DNA regions enriched in nucleosomes


#### Statistics

Gaussian distribution of the log2 ratio of the background signal to the enriched signal (a.k.a. “peak calling”) [^5]. 



[^1]: Giresi, P. G., Kim, J., McDaniell, R. M., Iyer, V. R., & Lieb, J. D. (2007). FAIRE (Formaldehyde-Assisted Isolation of Regulatory Elements) isolates active regulatory elements from human chromatin. Genome research, 17(6), 877-885.

[^2]: Davie, K., Jacobs, J., Atkins, M., Potier, D., Christiaens, V., Halder, G., & Aerts, S. (2015). Discovery of transcription factors and regulatory regions driving in vivo tumor development by ATAC-seq and FAIRE-seq open chromatin profiling. PLoS genetics, 11(2), e1004994.

[^3]: Waki, H., Nakamura, M., Yamauchi, T., Wakabayashi, K. I., Yu, J., Hirose-Yotsuya, L., ... & Fujita, T. (2011). Global mapping of cell type–specific open chromatin by FAIRE-seq reveals the regulatory role of the NFI family in adipocyte differentiation. PLoS genetics, 7(10), e1002311.

[^4]: Murtha, M., Strino, F., Tokcaer‐Keskin, Z., Sumru Bayin, N., Shalabi, D., Xi, X., ... & Dailey, L. (2015). Comparative FAIRE‐seq Analysis Reveals Distinguishing Features of the Chromatin Structure of Ground State‐and Primed‐Pluripotent Cells. Stem Cells, 33(2), 378-391.

[^5]: Buck, M. J., Nobel, A. B., & Lieb, J. D. (2005). ChIPOTle: a user-friendly tool for the analysis of ChIP-chip data. Genome biology, 6(11), R97.



